phone_verify/backends/nexmo.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+# Third Party Stuff
+import nexmo
+
+# Local
+from .base import BaseBackend
+
+
+class NexmoBackend(BaseBackend):
+
+    def __init__(self, **options):
+        super(NexmoBackend, self).__init__(**options)
+
+        # Lower case it just to be sure
+        options = {key.lower(): value for key, value in options.items()}
+        self._key = options.get("key", None)
+        self._secret = options.get("secret", None)
+        self._from = options.get("from", None)
+
+        self.client = nexmo.Client(key=self._key, secret=self._secret)
+
+    def send_sms(self, number, message):
+        self.client.send_message({
+            'from': self._from,
+            'to': number,
+            'text': message,
+        })
+
+    def send_bulk_sms(self, numbers, message):
+        for number in numbers:
```

Comment:
Does Nexmo have a bulk message sending API?

Reply:
No, I haven't come across any such API. They have only a few API for SMS sending and all of them include sending one message from one `from` number to one `to` number.

For more reference, you may check out here https://developer.nexmo.com/api/sms

Reply:
okay, thanks!